scss/components/_case-body.scss

```diff
@@ -8,6 +8,29 @@
   }
 }
 
+.civicase__case-body__not-found {
+  background: $crm-white;
+  color: $crm-gray-matte;
+  font-size: 14px;
+  font-weight: $font-weight-heading;
+  height: 100%;
+  padding: 18px 0;
+  text-align: center;
+
+  .material-icons {
+    color: $crm-gray-matte;
+    font-size: 64px;
+  }
+
+  .btn-danger {
+    margin-top: 20px;
+
+    .fa {
+      margin-right: 5px;
+    }
+  }
+}
+
```

Comment:
@deb1990 there are two issues:

1 - we are using too many custom values, we should stick to values defined in Shoreditch or Civicase.
2 - Ideally we should stick to using Bootstrap components. Isn't there one that we can already use for this?